src/components/Modal/index.js

```diff
@@ -31,6 +31,8 @@ class Modal extends React.Component {
 
   getClassName() {}
 
+  getId() {}
+
```

Comment:
You don't need to declare this method into the constructor if you are not gonna have a default. 
It'd be enough if you have this method into the child.